### ISSUE TITLE
My audit submissions: update placeholder page description

### ISSUE THREAD

**Issue by melchoyce** [GSA-TTS/FAC#763]
This issue covers the description of the "[my audit submissions](https://www.figma.com/file/1bNncVt1TcFFDhQslqFC2o/FAC-Playground?node-id=4954%3A24137&t=aHejgvyBoCtgtayE-4)" page.

![image](https://user-images.githubusercontent.com/2846578/215601608-dc6e2637-c06f-494f-a743-7c329a9784d6.png)

### Background

The former copy for this section assumed we'd launch the audit submission flow before the dissemination flow:

> The following audit submissions are only for 2022. You can find older audit submissions on the U.S. Census Bureau’s FAC website. 

However, it didn't touch on what the page is used for, so we've added in some placeholder copy that will need to be revised.
  
### Tasks

- [ ] Update copy
- [ ] Peer review

### Definition of Done

- [ ] Screens in Figma are ready to be handed off to front end

**Comment by Leighdiddy** [comment]
This page is currently for new and in progress audit submissions that have been assigned to a singular owner/email address. We may be showing completed submissions on this page as well TBD, which is why the copy is on hold.

**Comment by lauraherring** [comment]
Page purpose: 

- Supposed to be showing all audit submissions assigned to individual user, could be ones user created or ones assigned to user by other auditors/auditees. These are linked to user by user email address.
- Currently only for submissions in progress, cannot show previously submitted audits at this time.
- Also the page to begin a new audit submission, this should remain secondary on the page so users first confirm there isn't an open audit submission assigned to them and waiting on their involvement
- page title of "My Audit Submissions" has confused some users during testing, for auditors, they don't see these as "theirs", we've changed this public facing but may need to revisit internally

Open Questions
- are T&C confirmed now that FAC is under GSA and not Census? (Erin would know the current status)
- are we going to only display submissions in progress or will this page also display previously submitted/completed submissions?

**Comment by Leighdiddy** [comment]
"My Audit Submissions" = wording pain point with auditors, "my" is confusing because they don't consider the audits for themselves.

**Comment by lauraherring** [comment]
New copy for Figma:

In-progress Audit Submissions
The audit submissions below are in progress and assigned to your email address. These are for 2022 only. Completed audits from 2021 and earlier can be found on the U.S. Census Bureau's FAC website.

Create a new audit
Before you begin a new audit submission, check the list above to make sure one isn't currently in progress and assigned to your email address.

CC @melchoyce @austinhernandez @Leighdiddy

**Comment by mogul** [comment]
Leigh is going to deal with tornadoes and then get back to this.

**Comment by Leighdiddy** [comment]
Updated! Handing off to dev 👍

**Comment by JeanMarie-TTS** [comment]
@lauraherring , The year mentioned in the latest copy seems incorrect. GSA-FAC will accept audits for 2023. Census-FAC will continue to accepts audits for 2022.

**Comment by lauraherring** [comment]
Note: the magic words are "audit year 2022". New copy forthcoming.

**Comment by JeanMarie-TTS** [comment]
Laura,
Just to clarify ...
GSA-FAC will be accepting submissions for audit year 2023. Census-FAC will
accept submissions for audit year 2022 and earlier.
Will wait for you to update the copy in GH.

**Comment by thebestsophist** [comment]
We're gonna start collecting AY23 in September/October and then eventually pick up late AY22 submissions/resubmissions a few months later.
